src/adapters/adapter.js

```diff
@@ -100,6 +100,14 @@ export default class Adapter extends Configurable {
     return name;
   }
 
+  getAdapterUserIdById(userId) {
+    const botUser = this.bot.users.botUsers[userId];
```

Comment:
as discussed, we may want to make it so that you can't access bot users directly like this, an accidentally make changes - `const botUser = this.bot.getUser(userId)`